code/modules/chemistry/Reagents-PoisonEtc.dm

```diff
@@ -1408,39 +1408,57 @@ datum
 			fluid_r = 210
 			fluid_g = 180
 			fluid_b = 25
-			depletion_rate = 0.3
+			depletion_rate = 0.2
```

Comment:
Undocumented and unjustified.

Reply:
Oh, that probably came from me using the 0.2 version on my testing early on. To be honest i pretty much forgot about it and based a lot of the rest of the changes on it after, and caused it to be overall slower in the 10 to 20 unit bracket because of it. Do you think it better if i revert the depletion and make it slightly faster to maintain that 12 to 15 units to 300 blood level or keep as is?

Reply:
Trying to keep the balancing similar with a 0.3 depletion version, shouldn't be much of a change if i get it right. Servers me for starting changes from previous ones without checking very well if i did anything else to the code beforehand.